content/semver trick.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "Semver Trick"
```

Comment:
For consistency, I believe this should either be named "Semver trick", or "Dtolnay trick" should be renamed to "Dtolnay Trick".